tests/hpc/ww4_await_pxe_install.pm

```diff
@@ -11,24 +11,22 @@
 use Mojo::Base qw(hpcbase), -signatures;
 use testapi;
 use lockapi;
+use mmapi;
 use utils;
 use mm_tests;
 use POSIX 'strftime';
 
 sub run ($self) {
-    my $i = 0;
-
-    while ($i < 10) {
-        check_screen('pxe-start');
-        record_info "pxe", "$i";
-        sleep(1);
+    if (check_screen('pxe-start')) {
         send_key "ctrl-B";
-        sleep(20);
-        type_string("reboot", lf => 1);
         save_screenshot();
-        $i++;
+        record_info "pxe", "wait for controller";
     }
-
+    mutex_wait "ww4_ready";
+    barrier_wait('WWCTL_READY');
+    record_info 'WWCTL_READY', strftime("\%H:\%M:\%S", localtime);
+    type_string("reboot", lf => 1);
+    save_screenshot();
     while (1) {
         unless (check_screen('ww4-ready')) {
```

Comment:
I think you can also remove this loop and just use `assert_screen 'ww4-ready', 180;`. If I understand, loop just checks screen every second till we get to login screen.  We don't screenshots or `record_info` from this loop.

Reply:
Done. https://aquarius.suse.cz/tests/16636#step/ww4_await_pxe_install/8